BUILD

```diff
@@ -493,6 +493,34 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "grpc++_binder_hdr",
```

Comment:
This is for corresponding `.cc` targets to depends on so they can implement the header